### PR TITLE
Add popover when onhover line

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

**Review by ayolab — APPROVED** [review]

**Review by Walid-Sahnoun — APPROVED** [review]

File: src/components/tooltips/line-popover.js [hunk 1]
```diff
@@ -36,22 +36,28 @@ const useStyles = makeStyles((theme) => ({
...
     useEffect(() => {
-        if (lineId !== '') {
+        if (lineInfos) {
+            setLineInfo(lineInfos);
+        } else if (lineId && lineId !== '') {
             fetchLine(studyUuid, currentNode.id, lineId).then((value) => {
                 setLineInfo(value);
             });
-        } else {
-            setLineInfo(null);
```
Comment (AbdelHedhili) [inline src/components/tooltips/line-popover.js:52]: why did you remove that part ? I think it's good practice to empty the state when the data is no longer relevant so that we don't accidentally print old data for a few frames (even if there are others controls that prevents it)

**Review by AbdelHedhili — CHANGES_REQUESTED** [review]

**Review by AbdelHedhili — APPROVED** [review]
